students/idcrickmore/lesson03/mailroom.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+
+donors = [
+        ["Galileo Galilei", 348, 8377, 123],
+        ["Giovanni Cassini", 209],
+        ["Christiaan Huygens", 9135, 39],
+        ["Edmond Halley", 399, 1100, 357],
+        ["Edwin Hubble", 1899]
+        ]
+
+
+def menu():
+    print("\n------------------ MENU ------------------\n")
+    print("PLEASE CHOOSE FROM THE FOLLOWING THREE OPTIONS\n")
+    print("1. Send Thank You\n2. Create a Report\n3. Quit\n")
+
+    # the 'option' function calls an appropriate function based on user input
+    def options():
+        # save user input to 'choice' as a lowercase string
+        choice = input("-> ").lower()
+        # user input can be a number or a string
+        # if its a string, just look at the first letter
+        if choice[0] == '1' or choice[0] == 's':
+            thankyou()
+        elif choice[0] == '2' or choice[0] == 'c':
+            report()
+        elif choice[0] == '3' or choice[0] == 'q':
+            print("quiting")
+            quit()
+        else:
+            # if the user input does not match one of the 3 options,
+            # call the menu function again
+            menu()
+
+    options()
+
+
+def thankyou():
+    global donors
+    print("\n------------------ THANK YOU ------------------\n")
+    print("- type 'list' to to see a complete list of donors -")
+    print("- type 'menu' at any time to return to the menu -\n")
+
+    # simple function to list all the donors
+    # calls the 'thankyou' function at the end to ask for user input
+    def list():
```

Comment:
be careful about overwriting built-in objects, here you just overwrote `list` (notice github highlighted it different color)